src/common/translations/en.json

```diff
@@ -319,6 +331,7 @@
     "pending": "Pending",
     "done": "Done",
     "share": "SHARE",
+    "check": "CHECK",
```

Comment:
This string was added on other translations files?

Reply:
Fixed